distribution/lib/Standard/Visualization/0.0.0-dev/src/Table/Visualization.enso

```diff
@@ -59,11 +59,27 @@ prepare_visualization y max_rows=1000 = if y.is_error then (make_json_for_error
    Column Limit
 max_columns = 250
 
-## PRIVATE
+##PRIVATE
+leading_trailing_whitespace_count : Column -> Integer | Nothing
+leading_trailing_whitespace_count col =
+    find_whitespace col = 
+        filtered = col.to_vector.filter (c-> c.is_a Text && c.is_empty.not && (c.first.is_whitespace || c.last.is_whitespace))
+        filtered.length
+    if (col.value_type == Value_Type.Mixed || col.value_type.is_text) then find_whitespace col else Nothing
+
+##PRIVATE
+contains_atleast_one_in_list : Text -> Vector -> Boolean
+contains_atleast_one_in_list str list =
+    does_contain_list = list.map (char-> str.contains char)
+    filtered = does_contain_list.filter c->c
+    filtered.length > 1
```

Comment:
The name of this method suggest to me that the condition should be `>= 1` ("at least one").  Currently the `> 1` condition suggests rather "more than one". 

Please let's make the condition and the name consistent as otherwise it is misleading.